backend/ee/danswer/db/user_group.py

```diff
@@ -103,6 +119,42 @@ def fetch_user_groups_for_user(
     return db_session.scalars(stmt).all()
 
 
+def select_documents_by_usergroup(
```

Comment:
same comment here so the name implies that it returns a stmt rather than the result